shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java

```diff
@@ -48,28 +64,86 @@ public String getCommandName() {
   public void run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     AlluxioURI root = new AlluxioURI(args[0]);
+    checkConsistency(root, cl.hasOption("r"));
+  }
+
+  private void checkConsistency(AlluxioURI path, boolean repairConsistency) throws
+      AlluxioException, IOException {
+    //load the Newly created file's metadata in alluxio
+    ListStatusOptions listOptions = ListStatusOptions.defaults();
+    listOptions.setLoadMetadataType(LoadMetadataType.Always);
+    mFileSystem.listStatus(path, listOptions);
     CheckConsistencyOptions options = CheckConsistencyOptions.defaults();
-    List<AlluxioURI> inconsistentUris = FileSystemUtils.checkConsistency(root, options);
-    if (inconsistentUris.isEmpty()) {
-      System.out.println(root + " is consistent with the under storage system.");
+    List<AlluxioURI> inconsistentUris = FileSystemUtils.checkConsistency(path, options);
+    if (!repairConsistency) {
+      if (inconsistentUris.isEmpty()) {
+        System.out.println(path + " is consistent with the under storage system.");
```

Comment:
optional: if you `return` in the if branch, you can avoid the indentation of the else branch